File: spring-web/src/main/java/com/example/springwebex/model/pagenation/Response.java
package com.example.springwebex.model.pagenation;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import com.fasterxml.jackson.annotation.JsonInclude;
import com.fasterxml.jackson.annotation.JsonInclude.Include;

import lombok.Data;

@Data
@JsonInclude(Include.NON_NULL)  // serialize
@JsonIgnoreProperties(ignoreUnknown = true) // deserialize
public class Response {
    ResponseHeader header;
    ResponseBody body;
}

File: spring-web-kotlin/src/main/kotlin/com/example/springwebex/model/pagenation/Response.kt
package com.example.springwebex.model.pagenation

import com.fasterxml.jackson.annotation.JsonIgnoreProperties
import com.fasterxml.jackson.annotation.JsonInclude

@JsonInclude(JsonInclude.Include.NON_NULL)
@JsonIgnoreProperties(ignoreUnknown = true)
data class Response(
    var header: ResponseHeader? = null,
    var body: ResponseBody? = null
)
